appsec/tests/integration/src/main/groovy/com/datadog/appsec/php/mock_agent/rem_cfg/Capability.groovy

```diff
@@ -30,7 +30,9 @@ enum Capability {
     ASM_RASP_NOSQLI(27),
     ASM_RASP_XSS(28),
     APM_TRACING_SAMPLE_RULES(29),
-    CSM_ACTIVATION(30)
+    CSM_ACTIVATION(30),
+    ASM_DD_MULTICONFIG(42),
```

Comment:
You declare this new capability, but I see no other changes indicating that this is supported (and no tests).

Reply:
I did change te test which checks capabilities on appsec/tests/integration/src/test/groovy/com/datadog/appsec/php/integration/RemoteConfigTests.groovy

Reply:
Can you give me another test example using other of those?